releasenotes/notes/influxdb-default-retention-policy-aaa446ebe6fc3cb5.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Support default_retention_policy config for InfluxDB for new
+    database that are created. Falls back to infinite retention as
+    before when unspecified. Only applicable when db_per_tenant is
+    enabled. Retention policy for existing databases are not altered.
+    Must be specified in format such as 1w, 1m, 1y, etc.
```

Comment:
This is no longer true - but it doesn't really matter apart from upstream

Reply:
Oops! Fixed.